src/patients/related-persons/NewRelatedPersonModal.tsx

```diff
@@ -49,11 +51,17 @@ const NewRelatedPersonModal = (props: Props) => {
               searchAccessor="fullName"
               placeholder={t('patient.relatedPerson')}
               onChange={onPatientSelect}
+              isInvalid={isRelatedPersonInvalid}
               onSearch={async (query: string) => PatientRepository.search(query)}
               renderMenuItemChildren={(patient: Patient) => (
                 <div>{`${patient.fullName} (${patient.code})`}</div>
               )}
             />
+            {isRelatedPersonInvalid && (
+              <div className="text-left ml-3 mt-1 text-small text-danger invalid-feedback d-block">
+                {t('patient.relatedPersons.error.relatedPersonRequired')}
+              </div>
+            )}
```

Comment:
This change is good for now! However, this should really be functionality of the `Typeahead` component. Would you be willing to implement this change for https://github.com/HospitalRun/components/issues/349?